packages/cx-navigation/doc/Menu.md

```diff
@@ -22,27 +25,97 @@ In the end, the menu is rendered in the following structure:
 </Nav>
 ```
 
-The Wrapper is just an `<ul>` element. By using `asBodilessMenu()`, the default MenuClean provides an editable Item list, rendered as `<li>`. The Title inside is just an editable link in the clean component but the base/default tokens provided will replace it with CX MenuTitle.
+The Wrapper is simply an `<ul>` element. By using `asBodilessMenu()`, the default `MenuClean`
```

Comment:
simply?

Reply:
Whoops. 😅 
I remember trying this, and not liking it; didn't realize I had left it in.

✅ Updated.